tests/endtoend/test_cosmosdb_functions.py

```diff
@@ -16,7 +16,13 @@ def get_script_dir(cls):
     def test_cosmosdb_trigger(self):
         time.sleep(5)
         data = str(round(time.time()))
-        doc = {'id': 'cosmosdb-trigger-test', 'data': data}
+        doc = {'id': 'cosmosdb-trigger-test',
+               'data': data,
+               "_rid": "dummy12344",
+               "_self": "7U4=/docs/gpU4AJcm7U4KAAAAAAAAAA==/",
+               "_etag": "000-0500-0000-62598ff00000",
```

Comment:
Why add these values when its not asserted?

Reply:
they are asserted in line44-50

Reply:
these fields are populated/updated by cosmosdb every time document is inserted/updated, we can only assert their presence, i removed passing these fields in the input document as they should be auto populated.